packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -70,6 +71,10 @@ function useTranslationQueries(language: string, verseId: string) {
     { enabled: !!selectedLanguage }
   );
 
+  const bookProgressQuery = useQuery(['language-progress', language], () =>
+    apiClient.languages.findProgress(language)
+  );
```

Comment:
We'll want to add a custom api route to get the progress for a specific book. This query is going to be very slow because it has to load the entire Bible. We can also move this into the progress bar itself since we aren't using it elsewhere

Reply:
Done.